src/bundle/lookup.js

```diff
@@ -0,0 +1,42 @@
+define([
+	"../core/likely_subtags",
+	"../core/remove_likely_subtags",
+	"../core/subtags",
+	"../util/array/for_each"
+], function( coreLikelySubtags, coreRemoveLikelySubtags, coreSubtags, arrayForEach ) {
+
+	/**
+	 * bundleLookup( minLanguageId )
+	 *
+	 * @Cldr [Cldr class]
+	 *
+	 * @cldr [Cldr instance]
+	 *
+	 * @minLanguageId [String] requested languageId after applied remove likely subtags.
+	 *
+	 * Ref: http://goo.gl/qpwZ4n
+	 */
+	return function( Cldr, cldr, minLanguageId ) {
+		var availableBundleMap = Cldr._availableBundleMap,
+			availableBundleMapQueue = Cldr._availableBundleMapQueue;
+
+		if ( availableBundleMapQueue.length ) {
+			arrayForEach( availableBundleMapQueue, function( bundle ) {
+				var existing, maxBundle, minBundle, subtags;
+				subtags = coreSubtags( bundle );
+				maxBundle = coreLikelySubtags( Cldr, cldr, subtags, { force: true } ) || subtags;
+				minBundle = coreRemoveLikelySubtags( Cldr, cldr, maxBundle );
+				minBundle = minBundle.join( Cldr.localeSep );
+				existing = availableBundleMapQueue[ minBundle ];
+				if ( existing && existing.length < bundle.length ) {
+					return;
```

Comment:
I'm pretty sure we don't cover this in the style guide, but I agree that we should have an explicit return value here since a value is expected. I personally usually handle this with `null`, but staying with `undefined` is fine.